com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/message/Messages.properties

```diff
@@ -1,4 +1,4 @@
-aboutMessage={0} {1} build {2}\r\n\r\nCopyright (C) 2014 Search Solution Corporation.\r\nAll rights reserved by Search Solution.\r\n\r\nHome page\: {3}\r\nCUBRID Database project\: {4}\r\nCUBRID Tools project\: {5}
+aboutMessage={0} {1} build {2}\r\n\r\nCopyright (C) 2016 CUBRID Corporation.\r\nAll rights reserved by Search Solution.\r\n\r\nHome page\: {3}\r\nCUBRID Database project\: {4}\r\nCUBRID Tools project\: {5}
```

Comment:
This part also seems to need to be revised.

Reply:
same as above